libs/components/forms/testing/src/modules/input-box/input-box-harness.ts

```diff
@@ -79,7 +80,18 @@ export class SkyInputBoxHarness extends SkyQueryableComponentHarness {
   }
 
   /**
-   * Whether the custom error is triggered.
+   * Gets the custom form error.
+   */
+  public async getCustomFormError(
+    errorName: string,
+  ): Promise<SkyFormErrorHarness> {
+    return await this.locatorFor(
+      SkyFormErrorHarness.with({ errorName: errorName }),
+    )();
+  }
+
+  /**
+   * Whether the custom form error is triggered.
    */
```

Comment:
consumers will now have access to `hasCustomFormError` (see function below the new one) and `getCustomFormError`. `getCustomFormError` is a helper function to give consumers access to the child `SkyFormErrorComponent`'s harness that we project into input box. We have similar patterns established for other child components.